docs/source/faq.rst

```diff
@@ -722,15 +722,17 @@ backend:
 
 .. literalinclude:: ../../examples/launch_height_config_json.py
    :caption: examples/launch_height_config_json.py
-   :lines: 72-75
+   :start-after: if not use_sagemaker_backend:
+   :end-before: else:
```

Comment:
This is better than the raw line numbers, but also might be somewhat brittle. Just interesting. Maybe in the future we could add special comments or something to lines of interest for docs, and start/end on those (but that might also open up new challenges)